object.go

```diff
@@ -79,7 +79,7 @@ type ObjectAPI interface {
 	Get(context.Context, path.Path) (ipld.Node, error)
 
 	// Data returns reader for data of the node
-	Data(context.Context, path.Path) (io.Reader, error)
+	Data(context.Context, path.Path, bool) (io.Reader, io.Reader, error)
```

Comment:
Usually we don't change interface funcs directly.. but it's fine if all the implementations change along with this change